stacks/ambassador/README.md

```diff
@@ -1,152 +1,82 @@
 ```
 
-1. Apply with `kubectl apply -f quote.yaml`.
+After you deploy the stack you will need to configure a few more resources for the Ambassador Ingress Controller to work:
 
-1. We have our deployment and it's exposed with a service, but we still cannot connect to it without telling Ambassador where it is.  This is where the [Mapping](https://getambassador.io/docs/latest/topics/using/intro-mappings/) comes into play.  With a mapping, we can use kube-dns to let Ambassador know exactly where to look for the Quote service when the right request comes in.
+- The [Listener](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/03-setup-ingress-controller/ambassador.md#step-2---defining-the-listener-for-ambassador-edge-stack) Resource is required to configure which ports the Ambassador Edge Stack pods listen on so that they can begin responding to requests.
+- The [Host](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/03-setup-ingress-controller/ambassador.md#step-3---defining-the-hosts-for-ambassador-edge-stack) Resource configures TLS termination for enablin HTTPS communication.
+- The [Mapping](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/03-setup-ingress-controller/ambassador.md#step-6---configuring-the-ambassador-edge-stack-mappings-for-hosts) Resouce is used to configure routing requests to services in your cluster.
 
-    ```yaml
-    ---
-    apiVersion: getambassador.io/v2
-    kind: Mapping
-    metadata:
-      name: backend
-    spec:
-      prefix: /backend/
-      service: quote
-    ```
+Please refer to the [Ambassador](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/03-setup-ingress-controller/ambassador.md) Ingress Controller tutorial, for more details about checking Ingress Controller deployment status and functionality.
 
-1. Apply the mapping with `kubectl apply -f quote-backend.yaml`.
+### Uninstalling
 
-1. Test that it works by using `curl -k https://{{AMBASSADOR_IP}}/backend/` in your terminal.  You should see something similar to the following:
+To uninstall Ambassador Ingress Controller, you'll need to have Helm 3 installed. Once install, run the following:
 
-    ```bash
-    {
-        "server": "bleak-kumquat-n9qg6ra1",
-        "quote": "Non-locality is the driver of truth. By summoning, we vibrate.",
-        "time": "2020-05-08T18:33:54.578661743Z"
-    }% 
-    ```
-    - The IP address can be obtained by running:
-      ```bash
-      kubectl get svc -n ambassador -o 'go-template={{range .status.loadBalancer.ingress}}{{print .ip "\n"}}{{end}}'
-      ```
+```bash
+helm uninstall edge-stack -n ambassador
+```
 
-### Upgrading
+followed by:
 
-To upgrade Ambassador Edge Stack, use the following:
-```
-kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml
-helm repo update && helm upgrade -n ambassador ambassador datawire/ambassador
+```bash
+kubectl delete ns ambassador
```

Comment:
I noticed that the namespace `emissary-system` is also created as part of this install, should we add this one to this as well?

Reply:
I have updated the uninstall part to clean up the namespace `emissary-system` but also the crd's that were installed